vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/occupancy_handler.rb

```diff
@@ -27,7 +27,7 @@ def process
           )
         else
           # TODO: Add error cases
-          raise NotImplementedError
+          Rails.logger.warn "Encountered Double-Scan Situation"
```

Comment:
I imagine it's going away this week, so I vote to merge (selfishly to un-block me). 😸